compactcert/worker_test.go

```diff
@@ -385,9 +385,17 @@ func TestLatestSigsFromThisNode(t *testing.T) {
 	// Wait for a compact cert to be formed, so we know the signer thread is caught up.
 	_ = <-s.txmsg
 
-	latestSigs, err := w.LatestSigsFromThisNode()
-	require.NoError(t, err)
-	require.Equal(t, len(latestSigs), len(keys))
+	var latestSigs map[basics.Address]basics.Round
+	var err error
+	for x := 0; x < 10; x++ {
+		latestSigs, err = w.LatestSigsFromThisNode()
+		require.NoError(t, err)
+		if len(latestSigs) == len(keys) {
+			break
+		}
+		time.Sleep(256 * time.Millisecond)
```

Comment:
I would rather seeing this as an exponential backoff, but it's functionally correct as is.